docs/developer-docs/smart-contracts/advanced-features/serving-http-request.mdx

```diff
@@ -157,7 +147,7 @@ const HttpRequest = Record({
 });
 
 export default class{
-    http_request: @query([HttpRequest], HttpResponse, (req) => {
+    @query([HttpRequest], HttpResponse, (req) => {
```

Comment:
I'll just leave this as a general comment, in JavaScript the class syntax requires a method name and body, I left a couple examples above